test/parallel/test-crypto-sign-verify.js

```diff
@@ -11,9 +11,9 @@ const crypto = require('crypto');
 const fixtures = require('../common/fixtures');
 
 // Test certificates
-const certPem = fixtures.readSync('test_cert.pem', 'ascii');
-const keyPem = fixtures.readSync('test_key.pem', 'ascii');
-const modSize = 1024;
+const certPem = fixtures.readKey('rsa_cert.crt');
+const keyPem = fixtures.readKey('rsa_private.pem');
+const keySize = 2048;
```

Comment:
Thanks for pointing this out. @nodejs/crypto should we do this? Its not strictly a "move". I don't know if we really need to be testing keys so small. `1024` could be historical, or perhaps its important that we don't accidentally break small-key support for existing users.